### ISSUE TITLE
Review /legal to ensure it works with the new pricing

### ISSUE THREAD

**Issue by vhamer** [CartoDB/RevOps#52]
Hey @agnesgyorfi -

@vdiego (very bravely!) gave a detailed walkthrough of the [new pricing](https://docs.google.com/spreadsheets/d/1jk3UkOEWZ1LpKmIgBNBmNimjiRt_wZU4ibgK9p0piBQ/edit#gid=1818292216) to the NA Sales team just now. 

I know we've been trying to avoid a pricing scheme that would necessitate any kind of significant review of our terms (partly because we'll have to have the terms at carto.com/legal work both for all our legacy customers and their plans and simultaneously for this new pricing system). But some points during the discussion have opened my eyes...I think we'll need to do a review of our terms to ensure we're

One example. The pricing under consideration includes specific levels of Map Views per month. Jatorre spoke up to explain that our definition of Map Views is changing from the previous "15 tiles loaded in a browser window" to "each call to the Maps API." Apparently Google and Mapbox have switched to similar definitions, so we're following suit. So that definition needs to change in our LDS Terms: https://drive.google.com/file/d/1bDN0VeKg7125JK281maoUUGtHLELYNot/view

Side note that we should use the term Map Loads rather than Map Views.

Another example. Our current [Platform Terms](https://drive.google.com/file/d/1Uw3Cz9IU50K1W6nARwAwZIf9BE-f_dOz/view), section 5.c., indicates that unless a customer has purchased the right to use CARTO in a Commercial Product, they do not have a license to use CARTO in anything besides Internal and Non-commercial Applications. But the new pricing says 'if you buy a package, you can use it to power both Internal and Public applications...you're not limited by license, you're only limited by the number of Map Views available in your CARTO Package.

Related, each of the new packages has various limits that are based on the package size. We need to figure out which of those to list in the product description on the Quotes, which ones to include in the Usage Limits page of our website (which is incorporated by reference into the terms).

Basically I think there's a need to do a full review of the pricing, a full review of the terms, a full review of all the historical plans, and make sure we don't end up in 'oops we can't enforce that limit legally, because we didn't write it down' land.

(cc @ldieye @ruipperez for awareness)
